### PR TITLE
Fix #8110: Use a single name for the title sequences directory

### LINKED ISSUE #8110: Three different folder names are used for title sequences
This is something that stood me out when writing a tutorial about the user folder. The precreated folder in the user's directory is named `sequence`. The default ones are in `title`, and `GetUserSequencesPath()` uses `title sequences`.

This makes no sense. It should use one folder name. I propose using `sequence`.

### PR DESCRIPTION
**Pull request by Gymnasiast**
Review note: I would appreciate a good look at how I handled strings.

I have checked the x86_64 MSVC, Xcode and x86 Linux artifacts. These all contain the title sequences, and in the correct place.

### CODE REVIEW

**Comment by IntelOrca** [comment]
So will players need to move their title sequences into a different folder now?

**Comment by Gymnasiast** [comment]
Yes.

File: CMakeLists.txt [hunk 1]
```diff
@@ -299,14 +299,14 @@ list(APPEND DOC_FILES "${ROOT_DIR}/contributors.md"
 # targets, like `install`, so we have to trick it and execute dependency ourselves.
 install(CODE "execute_process(COMMAND \"${CMAKE_COMMAND}\" --build \"${CMAKE_CURRENT_BINARY_DIR}\" --target g2)")
 if (DOWNLOAD_TITLE_SEQUENCES)
-    # If openrct2.parkseq or data/title/ exists, assume all the title sequences are already present
+    # If openrct2.parkseq or data/sequence/ exists, assume all the title sequences are already present
     install(CODE
-        "if (EXISTS \"\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/title/openrct2.parkseq\" OR EXISTS ${CMAKE_SOURCE_DIR}/data/title/)\n\
+        "if (EXISTS \"\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/sequence/openrct2.parkseq\" OR EXISTS ${CMAKE_SOURCE_DIR}/data/sequence/)\n\
```
Comment (janisozaur) [inline CMakeLists.txt:304]: The repo with sequences needs to be updated

Comment (Gymnasiast) [reply CMakeLists.txt:304]: Hm, why? I tried several artifacts and they all appeared to properly contain the title sequences.

**Pull request merged**